### DISCUSSION TITLE
Is there any JSON API?

### DISCUSSION THREAD

**Discussion by pbuzdin** [datenstrom/community#439] [Ask a question] [answered]
Is there any kind of JSON REST API?
Will it be in future?

**Comment by schulle4u** [comment, marked as answer]
Currently there's no JSON API available. What would be the benefit of such an API for Yellow users or developers? 
In my opinion this is bayond Yellow's scope, but we are always open for new ideas.  😀

**Comment by pbuzdin** [comment]
For example:
Using Yellow as backend and GUI for JSON API.
Utilizing in mobile app, JS app or in third-party integration.
